README.md

```diff
@@ -65,10 +65,13 @@ Include the Proguard mapping file to deobfuscate Android performance traces:
     api-key: ${{ secrets.MOBILE_DEV_API_KEY }}
     name: ${{ github.sha }}
     app-file: app.zip
+    mapping-file: app.dSYM.zip
```

Comment:
As far as I know, the `.zip` is not needed because we automatically zip when it is a directory

Reply:
good point , I don't see any zip extensions on the Android part. I'd also remove it from the `app-file`

cc @dmitry-zaitsev

Reply:
That is correct, zip is not needed - action takes care of that